website_docs/manual_instrumentation.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Ruby Manual Instrumentation"
+weight: 24
+description: >
+  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Ruby_SDK.svg"></img>
+  A language-specific implementation of OpenTelemetry in Ruby.
+---
+
+## Adding Manual Instrumentation
+
+Auto-instrumentation is the easiest way to get started with instrumenting your code, but in order to get the most insight into your system, you should add manual instrumentation where appropriate.
```

Comment:
I've been experimenting with phrasing manual instrumentation added to auto-instrumentation as "enriching auto-instrumentation" to distinguish the use case from "I want _only_ my bespoke artisanal manual spans." My hypothesis is that identifying which of those two use cases a reader thinks may apply to them routes them down a setup path that pays off for them sooner than the other. My further guess is that auto-inst-with-manual-enrichment is the happier path for most users. If this seems like a promising perspective, maybe there are some tweaks we can make to this page to help readers down one of the two paths?

Reply:
Perhaps we could create a separate section that describe use cases and patterns or we could set aside time to co-author our book series "Head First OpenTelemetry", "OpenTelemetry in Action", and "OpenTelemetry the Hard Way".

Reply:
I did make a few changes. Let me know if it flows a little better.